### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by souvikinator**
- Added reference image to make clear what this tool does.
- Added installation process using Makefile

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # anonymousface
 
+## Before
+
+<img src="https://imgur.com/Xc5918M.jpg" width="25%" style="display:inline;">
```
Comment (mattn) [inline README.md]: Please put images on this repository.

File: README.md [hunk 2]
```diff
@@ -14,6 +22,18 @@ $ anonymousface < input.jpg > output.jpg
...
+### Using make file
+
+```bash
+git clone https://github.com/mattn/anonymousface.git && cd anonymousface
+```
+
+and then
+
+```bash
+make
```
Comment (mattn) [inline README.md]: Makefile is not provided for the users. This is for maintener. (me)

File: README.md [hunk 3]
```diff
@@ -23,3 +43,7 @@ MIT
 ## Author
 
 Yasuhiro Matsumoto (a.k.a. mattn)
+
+```
+
+```
```
Comment (mattn) [inline README.md]: Needless.

File: README.md (hunk 2 above)
Comment (souvikinator) [reply README.md]: alright, so should I add installation instruction using `go install` as the current instruction has some issue?

**Comment by mattn** [comment]
Thanks but I don't like this images.

**Comment by souvikinator** [comment]
Okay so you can provide the image and I can update?

**Comment by mattn** [comment]
Could you please include fix only installation instruction?

**Comment by souvikinator** [comment]
yeah sure, no problem.

**Review by mattn — APPROVED** [review]

**Comment by mattn** [comment]
Thank you.

**Pull request merged**